.travis.yml

```diff
@@ -37,13 +41,16 @@ before_script:
  - "if [ \"$BEHAT_TEST\" = \"\" ]; then php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss; fi"
  - "if [ \"$BEHAT_TEST\" = \"1\" ]; then php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/behat-extension; fi"
  - cd ~/builds/ss
+ - "if [ \"$CMS_TEST\" = \"1\" ]; then composer require --prefer-dist silverstripe/cms:$CORE_RELEASE.x-dev"
```

Comment:
`master-x.dev` :D just a note we'll need to change this on master, heh.

Does self.version work here?